pkgs/development/python-modules/androguard/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, buildPythonPackage, fetchPypi, future, networkx, pygments, lxml, colorama, matplotlib,
+  asn1crypto, click, pydot, ipython, pyqt5, pyperclip }:
+
+buildPythonPackage rec {
+  version = "3.3.3";
+  pname = "androguard";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1zlmn3byh2whg7k2xmcd7yy43lcawhryjnzcxr9bhn54709b6iyd";
+  };
+
+  propagatedBuildInputs = [
+    future
+    networkx
+    pygments
+    lxml
+    colorama
+    matplotlib
+    asn1crypto
+    click
+    pydot
+    ipython
+    pyqt5
+    pyperclip
+  ];
+
+  # The package doesn't contain any tests.
```

```suggestion
  # Tests are not shipped on PyPI.
```
See https://github.com/androguard/androguard/tree/master/tests.